packages/ocal/ocal.0.2.2/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Richard Mortier <mort@cantab.net>"
+authors: [ "Richard Mortier" ]
+license: "ISC"
+
+homepage: "https://github.com/mor1/ocal"
+dev-repo: "git+https://github.com/mor1/ocal.git"
+bug-reports: "https://github.com/mor1/ocal/issues"
+doc: "https://mor1.github.io/ocal/"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "astring"   {build}
+  "calendar"  {build}
+  "cmdliner"  {build}
+  "dune"      {build}
+  "notty"     {build & >="0.2.0"}
+  "ocaml"     {build & >="4.02.3"}
```

Comment:
mmh, something tells me `ocaml` shouldn't be a build dependency. What if `dune` decides to compile the executable to bytecode if the platform doesn't support native code?

Reply:
This is as recommended by `dune-release lint`:
```
: mort@greyjay:ocal#master= $; make lint
dune build @lint
dune-release lint
[FAIL] lint opam file ocal.opam:
       opam lint -s messages:
       /Users/mort/src/ocal/ocal.opam: Errors.
                    error 32: Field 'ocaml-version:' and variable 'ocaml-version' are deprecated, use a dependency towards the 'ocaml' package instead for availability, and the 'ocaml:version' package variable for scripts
[FAIL] lint /Users/mort/src/ocal failure: 1 errors.
make: *** [lint] Error 1
```
(Or at least, that's how I interpreted the error message! Am I supposed to do something else here?)

Reply:
oh sorry, I should have been more clear. I meant to insist on the `build` dependency part, not the `dependency` part of the sentence ^^

OCaml should of course be a dependency but not flagged as a `{build}` dependency probably. Anyone has a thought on that btw? @rjbou @rgrinberg?

Reply:
Setting `ocaml` as `{build}` seems wrong. At the very least, the compiler contains pervasives and it's highly unlikely that Pervasives are only used for building a package.